Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentImportMenu.swift

```diff
@@ -142,7 +142,11 @@ struct AttachmentImportMenu: View {
             if let presetFileName = newAttachmentImportData.fileName {
                 fileName = presetFileName
             } else {
-                let attachmentNumber = element.attachments.count + 1
+                // `element.attachments` is now marked `async throws`, so this
+                // needs to be modified. We can address it as part of
+                // Apollo #682. Temporarily, just use a random Int.
+//                let attachmentNumber = element.attachments.count + 1
+                let attachmentNumber = Int.random(in: 0...99)
```

Comment:
In #758, I accessed the attachments up above as apart of the first `guard`. Here's the [commit](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/758/commits/3a3da375eb25fe2c36d2ffca98403843ba6d867d). 

I'll close that PR out but would you just want to change this to do the same?